### ISSUE TITLE
WordNumber counts punctuation as a word

### ISSUE THREAD

**Issue by edwardwoodward** [redpen-cc/redpen#722]
Testing RedPen 1.7 on an English text file. The sentence below has 28 words, but WordNumber counts 31 and flags an error.

```
There is no real path, so first follow the line of the foot of the rocks past Kawa, then cut straight up to the next level of slabs.
```

It is counting the 2 commas and the period as words. It does the same with colons and semi-colons. I haven't tried with other punctuation. Incidentally, the documentation states that the max_num default value is 50, but in practice it appears to be 30. I will change the English manual accordingly.

**Comment by takahi-i** [comment]
Ref https://github.com/redpen-cc/redpen-doc/commit/2257a5f254139f8b122ab694585cde9263db8874

**Comment by takahi-i** [comment]
Hello @edwardwoodward! I fixed this problem in#723.
